### PR TITLE
Option to select a custom configuration when doing config deployment

### PR DESCRIPTION
**Pull request by rinsekloek**
* Option to select alternative Configuration 

Added an extra parameter when doing 'configure_routers' from the CLI. This extra parameyer gives you the option to select a specific template to be deployed. For example to update prefix-lists from IRR updates during a daily cron and not deploy the complete configuration template:

```
usage: manage.py configure_routers [-h] [--no-commit-check] [--limit [LIMIT]] [-t] [-c CONFIG] [--version] [-v {0,1,2,3}] [--settings SETTINGS] [--pythonpath PYTHONPATH]
                                   [--traceback] [--no-color] [--force-color] [--skip-checks]

Deploy configurations on routers.

optional arguments:
...
  -c CONFIG, --config CONFIG
                        Override default configuration to use for config generation/deployment. Give the name of the configuration.
...
```

* Example:

```
# python3 /opt/peering-manager/manage.py configure_routers -c "Nokia SROS Prefix lists update" --limit sros01
[*] Selecting configuration template 'Nokia SROS Prefix lists update'
[*] Deploying configurations
  - sros01 ... success
```

### CODE REVIEW

File: peering/management/commands/configure_routers.py [hunk 1]
```diff
@@ -25,8 +26,36 @@ def add_arguments(self, parser):
...
+            help="Override default configuration to use for config generation/deployment. Give the name of the configuration.",
+        )
+
+    def process(
+        self,
+        router,
+        quiet=False,
+        as_task=False,
+        no_commit_check=False,
+        config_override=False,
```
Comment (gmazoyer) [inline peering/management/commands/configure_routers.py:42]: If `config_override` is supposed to be a string containing the name of the config, the default value would probably better as `""` or `None`.

**Review by gmazoyer — CHANGES_REQUESTED** [review]
Check comment for minor change. The documentation related to the Nokia SROS template might be better in another PR dedicated to it. The PR looks good.

**Comment by rinsekloek** [comment]
I have changed the default to "" and removed the nokia-sros from the main branch. (My inted was that it was not included in this PR)

**Review by gmazoyer — APPROVED** [review]
LGTM

**Comment by rinsekloek** [comment]
> Check comment for minor change. The documentation related to the Nokia SROS template might be better in another PR dedicated to it. The PR looks good.

I have started with the Nokia SROS template in a seperate branch:
https://github.com/rinsekloek/peering-manager/blob/nokia-sros/docs/templating/examples/nokia-sros.md

**Pull request merged**